docs/ff-integrations/payments/braintree.md

```diff
@@ -0,0 +1,240 @@
+Make sure the user is authenticated before triggering the Braintree Payment Action, otherwise, it will result in an error. You can follow the steps on [**this page**](../authentication/firebase-auth/auth-initial-setup.md) to set up Authentication.
+:::
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://demo.arcade.software/BiF5CcHTJBcU2brcLVVT?embed&show_copy_link=true"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+#### Using Credit Card
+
+If you want to keep only the Credit Card option on your checkout page, you'll need to add the **CreditCardFrom** widget to the page. Follow the steps below:
+
+1. Select the **Payment Method** as ***Credit Card***.
+2. Drag and drop the [**CreditCardFrom**](../../resources/ui/components/built-in-components/creditcardform.md) widget onto the canvas.
+3. You can modify the design of the form widget as per your app's needs.
+4. Again select the **checkout button** to complete defining the Action.
+5. Enter the **Currency Code** and you can define the optional attributes like **Tax Rate Percentage** and **Shipping Cost**.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://demo.arcade.software/SMD1vtLJFXxG5AuJrgQS?embed&show_copy_link=true"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+### 4. Testing
+
+Braintree payments work on real Android devices or in emulators, and App Store purchases only work on real iOS devices. [This document](../../testing-deployment-publishing/running-your-app/local-run.md) has instructions on how to run your app on an Android or iOS device.
+
+:::info
+The Braintree Payments cannot be tested in Preview Mode, Test Mode, or Run Mode.
+:::
+
+To test your app before deployment:
+
+1. Download and run your project as described [here](../../testing-deployment-publishing/running-your-app/local-run.md).
+2. To test the purchase, you can use any of these [basic test card numbers](https://stripe.com/docs/testing#cards).
```

Comment:
This links to stripe doc. Is that intentional?

Reply:
Yes, they work for any payments provider.